### PR TITLE
Add CopyNumberVariant and abstract GenomicVariant class

### LINKED ISSUE #105: Add GenomicCopyNumber
Follow pattern of GenomicShortVariant and use this API: https://docs.us.lifeomic.com/api/#query-copy-number-data

### PR DESCRIPTION
**Pull request by rcdilorenzo**
Closes #105

### CODE REVIEW

**Review by mschroering — DISMISSED** [review]

File: phc/easy/query/api_paging.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def recursive_paging_api_call(
         max_pages = 1
 
     # Compute count and add to progress
-    if _count is None and len(_prev_results) == 0:
+    if _count is None and len(_prev_results) == 0 and False:
```
Comment (mjtieman) [inline phc/easy/query/api_paging.py:48]: Vestige from testing?

Comment (rcdilorenzo) [reply phc/easy/query/api_paging.py:48]: Ouch. Yes. That's when I filed the big report about the mismatched count. I'll fix.

File: phc/easy/abstract/genomic_variant.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+            auth_args=auth_args,
+            **test_args,
+        )
+        args["variant_set_ids"] = variant_set_ids = list(test_df.id)
+
+        if len(variant_set_ids) > MAX_VARIANT_SET_IDS and (
+            max_pages or (not all_results and page_size)
+        ):
+            print(
+                "[WARNING]: All result limit paramters are approximate when performing genomic data retrieval."
```
Comment (jonathanbranam) [inline phc/easy/abstract/genomic_variant.py:109]: typo: paramters => parameters

**Review by jonathanbranam — DISMISSED** [review]
Looks good to me, not that that means much. One minor typo in a print statement.

**Review by jonathanbranam — APPROVED** [review]

**Review by mjtieman — APPROVED** [review]

**Review by RyanRatcliff — APPROVED** [review]

**Pull request merged**